src/renderers/native/NativeMethodsMixin.js

```diff
@@ -212,7 +212,7 @@ function mountSafeCallback(
   callback: ?Function
 ): any {
   return function() {
-    if (!callback || (context.isMounted && !context.isMounted())) {
+    if (!callback || (typeof context.isMounted === 'function' && !context.isMounted())) {
```

Comment:
shouldn’t Flow understand this to be a `function` if it exists? https://github.com/facebook/flow/blob/master/lib/react.js#L74